### PR TITLE
Feature: Display, Add and Remove books via JavaScript

### PR DESCRIPTION
**Pull request by kelvinbush**
In this pull request we did the following:

- Set up html to render the list of books.
- Added a form for adding a book to the list.
- Added JavaScript to add, remove and render the list of books dynamically.
- Implemented saving the list of books to local storage.
- Added linters and documentation to the project.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -32,20 +32,20 @@ jobs:
...
+#      - uses: actions/checkout@v2
+#      - uses: actions/setup-node@v1
+#        with:
+#          node-version: "12.x"
+#      - name: Setup Stylelint
+#        run: |
+#          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+#          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.stylelintrc.json
+#      - name: Stylelint Report
+#        run: npx stylelint "**/*.{css,scss}"
```
Comment (Meltrust) [inline .github/workflows/linters.yml]: - [x] Kindly  remove this old piece of commented code, in order to improve the cleanliness of the repo 💪 
- [x] OPTIONAL: Helpful tip: Webhint and Lighthouse, are not required for the Javascript projects.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @kelvinbush,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- - CRUD works ✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — APPROVED** [review]
Hi @kelvinbush,


## Wow, you did it 🎉
Brilliant
## Thank you for deleting the commented out code, as requested in the previous review💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (IshmaelKargbo) [reply .github/workflows/linters.yml]: Thanks

**Pull request merged**